### DISCUSSION TITLE
Version 2.983

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1112] [General]
* Fix broken Youtube video loading
* Minor cleanups, and preparations for raising the required Android level. Old Androids will be supported through 'SkyTube Legacy'
* Updated translations for:
    * Azerbaijani - Nizami Semidov
    * Basque - Julio Montero
    * Chinese (Simplified) - Eric
    * Chinese (Traditional) - Jeff Huang
    * Croatian - Milo Ivir
    * Hungarian - Szia Tomi
    * Spanish - Alfredo Cruz
    * Polish - Michal L
    * Turkish - Oğuz Ersen
    * Ukrainian - Ihor Hordiichuk

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/v2.983'>Version 2.983</a>.</em>

**Comment by drogga** [comment]
@gzsombor Where are the OSS release builds ?
I think that you should reConfigure the android.yml workflow, or create a new one (named "release" and rename this current one to "snapshot" or "debug"), so this can be done automatically, instead of manually and maybe make the debug/snapshot builds be signed with your key/signature too, the same as on the release builds, instead of a debug one, which I think is with different hash every time (that prevents updates over one newer snapshot to old one), because it's generated on runtime, instead of pre-build one, that btw can be uploaded to the repo, your own key too, and can even be hidden if you like to and the decrypt key can be added to the workflow config .yml file(s).

Edit:
^This applies to the "Legacy" repo too, no OSS there too...
